tests/python/tir-transform/test_tir_transform_convert_ssa.py

```diff
@@ -350,7 +351,8 @@ def kernel_2(A: T.Buffer([256], "float32")):
         return mod
 
     def expected(self):
-        @I.ir_module
+        # complaints of duplicate definitions of threadIdx_x
```

Comment:
Can you verify that this occurs in the `expected` and not just the `before`?  The before case should be ill-formed, because `threadIdx_x` is deliberately defined outside the TIR function and reused, but the expected case defines `threadIdx_x` inside of each function.

Reply:
It happens in both